ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -2667,6 +2667,55 @@ AHU_BYPSSPC_DX3SC_ECON_HT2SC_SFSS_SFVSC:
   - SFSS
   - SFVSC
 
+#AU-SY-ODI types  
+AHU_DFFC_ZTC_CHWZTC_HWZTC_STM_RTM_CO2C_FDPM_AHAC:
+  description: "Single zone air handling unit with CO2 override and after hours override"
+  is_canonical: true
+  implements:
+  - AHU
+  - DFFC
+  - ZTC
+  - CHWZTC
+  - HWZTC
+  - STM
+  - RTM
+  - CO2C
+  - FDPM
+  - AHAC
+
+AHU_SFVSC_ZTC_RTM_STM_HPZTC_CWDT_CO2C_OADM_FDPM_AHAC_WLDS:
```

Comment:
if single zone (i.e. no downstream VAVs) then **discharge** is used instead of **supply** -- convert all things that implement **supply** (SFVSC and STM) to their discharge equivalents (DFVSC and DTM are the correct alternatives).

Reply:
Changes made.